keyboards/splitkb/aurora/sweep/keymaps/flinguenheld/features/oled.c

```diff
@@ -0,0 +1,118 @@
+#include "../font/logos.c"
```

Comment:
C source files should not be `include`'d. Include them via `SRC +=` in your rules.mk file.

Reply:
I'm getting crazy with includes ^^`
Is it correct to put them in the keymap.c ?